doc/source/examples/dem/sandpile-formation/sandpile-formation.rst

```diff
@@ -0,0 +1,305 @@
+        set friction coefficient particles    = 0.8
+        set rolling friction particles        = 0.3
+      end
+      set young modulus wall           = 4000000
+      set poisson ratio wall           = 0.49
+      set restitution coefficient wall = 0.7
+      set friction coefficient wall    = 0.8
+      set rolling friction wall        = 0.3
+    end
+
+
+Model Parameters
+~~~~~~~~~~~~~~~~
+
+.. code-block:: text
+
+    subsection model parameters
+      subsection contact detection
+        set contact detection method                = dynamic
+        set dynamic contact search size coefficient = 0.8
+        set neighborhood threshold                  = 1.3
+      end
+      set particle particle contact force method = hertz_mindlin_limit_overlap
+      set particle wall contact force method     = nonlinear
+      set integration method                     = velocity_verlet
+      set rolling resistance torque method       = epsd_resistance
+      set f coefficient                          = 0.0 
+    end
+
+.. note::
+
+  Only when the Elastic-Plastic Spring-Dashpot (epsd) rolling resistance model is selected should f be specified (f=0 here).
+  More information regarding the DEM Model parameters is given in the Lethe documentation, i.e. `DEM Model Parameters <../../../parameters/dem/model_parameters.html>`_.
+
+Particle Insertion
+~~~~~~~~~~~~~~~~~~
+
+Particles are inserted in an insertion box in the upper part of the hopper. In this simulation, the hopper is filled through 12 insertion steps.
+
+.. code-block:: text
+
+    subsection insertion info
+      set insertion method                               = volume
+      set inserted number of particles at each time step = 250
+      set insertion frequency                            = 10000
+      set insertion box points coordinates               = -0.34, 0.7 : 0.34, 1.2
+      set insertion distance threshold                   = 1.5
+      set insertion maximum offset                       = 0.1
+      set insertion prn seed                             = 20
+    end
+
+.. note::
+
+  Due partly to the bi-dispersed simulation, changing the prn seed to a different value can lead to different results.
+
+
+Simulation Control
+~~~~~~~~~~~~~~~~~~
+
+The simulation runs for 50 seconds of real time. We output the simulation results every 1000 iterations.
+
+.. code-block:: text
+
+    subsection simulation control
+      set time step        = 2e-5
+      set time end         = 50
+      set log frequency    = 1000
+      set output frequency = 1000
+      set output path      = ./output/
+    end
+
+.. note::
+
+  To compare with the results of J.Ai *et al.*, the end time should be set at 50s or at least 35s to see the progression of the curve. 
+  It can be reduced to 15s to see the fully formed sandpile but the height of the pile is only measured after 10s and may continue to decrease after 15s, particularly with the constant rolling resistance model.
```

Comment:
```suggestion
  It can be reduced to 15 s to see the fully formed sandpile but the height of the pile is only measured after 10 s and may continue to decrease after 15 s, particularly with the constant rolling resistance model.
```